TOMP-API.yaml

```diff
@@ -1251,25 +1249,25 @@ components:
           type: string
         geometry:
           $ref: '#/components/schemas/polygon'
-        opening-times:
+        openingTimes:
```

Comment:
I think this should be called returnTimes if that is what is meant. If opening hours are what is meant, those should derive from the station or system and not be reiterated here.